### ISSUE TITLE
[Regression]: PauseAsync/CodeGen no longer highlighting the exact location of locators on screen

### ISSUE THREAD

**Issue by Shehreem-Solirius** [microsoft/playwright-dotnet#2933]
### Last Good Version

1.39

### First Bad Version

1.40

### Steps to reproduce

Note: I have not created a repo as it's not needed for this issue, please see steps below

1. Run the following command in the terminal - npx playwright codegen https://guidance.submit-learner-data.service.gov.uk/
2. Click any locator on screen
3. Notice, the top left corner is highlighted rather than the exact location of the locator
<img width="1498" alt="image" src="https://github.com/microsoft/playwright-dotnet/assets/77390583/b06fcbd1-7e2d-4ed1-ad64-f7003f7274b3">

### Expected behavior

I expect the exact location of the locator to be highlighted, e.g. in the image above, the search bar

### Actual behavior

The top left corner of the screen is highlighed

### Additional context

This is annoying when trying to create custom locators
It also happens when running PauseAsync
This used to work before we updated playwright due to older versions not supporting dotnet8, so we cannot downgrade to get the functionality
This seems to only happen on our sites, Google for example, works fine

### Environment

```Text
- Operating System: Windows
- Browser: Chromium
- .NET Version (TFM): net8.0
```

**Comment by Shehreem-Solirius** [comment]
Amazing, thanks, looking forward to the update 😄

**Comment by mxschmitt** [comment]
I'll close it, since this gets fixed in 1.44.0 which we'll release shortly.
